lib/ecto_ldap/adapter.ex

```diff
@@ -44,6 +48,29 @@ defmodule Ecto.Ldap.Adapter do
     {:reply, search_response, state}
   end
 
+  def handle_call({:update, schema_meta, fields, filters}, _from, state) do
```

Comment:
gotcha, so are you suggesting an `update_options` map that contains `schema_meta`, `fields`, and `filters`?